### DISCUSSION TITLE
Wine Mono is not installed error - also may need MS visual C++

### DISCUSSION THREAD

**Discussion by metaphysician** [The-Wineskin-Project/WineskinServer#285] [Q&A]
this is very likely a really basic question but i used WS11 CX64bit21.2.0 and WS11 CX21.2.0 and tried installing the same app in 64 bit and 32 bit. i was able to run the installer successfully both times but when test running the app it completes with nothing running and then says 'Wine Mono Is Not Installed' in the log. i unchecked the 'Disable Mono Installation' on the 32 bit installation but it made no difference. nothing runs. so i need to find out how to install Wine Mono at least. i have been told by the dev that it needs the MS Visual C++ version installed as well. any help appreciated!

**Comment by emendelson** [comment]
Try this (I'm just guessing): run Wineskin, Advanced, Tools, Winetricks. Enter 

C++

in the search field, and select one of the vcrun options to install it. Maybe the dev can tell you which is best, but it can't hurt to install more than one.

This is just a guess!

**Reply by Gcenx** [reply]
vcrun2019 also includes vcrun2017 & vcrun2015 so usually that’s a good starting point for more modern applications.

**Comment by Gcenx** [comment]
wine-mono is wines reimplementation of Microsoft .Net you can install many versions of .Net from the winetricks menu.
